controllers/ticketController.js

```diff
@@ -0,0 +1,133 @@
+const {
+    Ticket
+} = require("../models/ticket.models");
+const {
+    validationResult
+} = require('express-validator');
+
+const {
+    errorHelpers
+} = require('../helpers/errors.helpers');
+const {
+    mongo
+} = require("mongoose");
+
+// api to create ticket
+exports.createTicket = async(req, res) => {
+    try {
+        validationResult(req).throw();
+        const {
+            title,
+            description,
+            ticket_raise_by,
+            priority,
+            status
+        } = req.body;
+
+        let ticket = await Ticket.create({
+            title,
+            description,
+            ticket_raise_by,
+            priority,
+            status
+        });
+
+        res.status(200).json({
+            ticket,
+        });
+
+    } catch (error) {
+        res.send(error);
+        //errorHelpers.catchError(error, res);
+    }
+
+};
+// update api
+exports.updateTicketById = async(req, res) => {
+    try {
+        validationResult(req).throw();
+        const updateData = {
+            title: req.body.title,
+            description: req.body.description,
+            ticket_raise_by: req.body.ticket_raise_by,
+            priority: req.body.priority,
+            status: req.body.status
+        };
+
+        await Ticket.update({
+            "_id": new mongo.ObjectID(req.params.id)
+        }, {
+            $set: updateData
+        }).then(result => {
+            res.status(200).json({
+                "status": 200,
+                "message": "Ticket Status Updated Successfully!",
+                "data": ""
+            });
+        }).catch(err => {
+            res.status(400).json({
+                "status": 400,
+                "message": err
+            });
+        });
+
+
+    } catch (error) {
+        res.send(error);
```

Comment:
Instead of sending error, please be more specific. The frontend (dev) usually needs the error message. 

Would be better if you consider using a specific notation everywhere. 

The responses should be in a form like this - 

`
{
  success: true/false,
  errorrMessage: 'Will be null if there is no error else it will be error.message',
  data: [...your response]
}
`

Reply:
Yes , all error and success message is now handled from a separate module